kitsune/sumo/utils.py

```diff
@@ -315,6 +319,42 @@ def get_browser(user_agent):
     return browser
 
 
+def has_blocked_link(data):
+    # TODO: deal with punycode when presented in non-ascii form
+    for match in POTENTIAL_LINK_REGEX.finditer(data):
+        tld = match.group(1).upper()
+        if tld in VALID_TLDS:
+            full_domain = match.group(0).lower()
+            in_allowlist = False
+            for allowed_domain in settings.ALLOW_LINKS_FROM:
```

Comment:
couldn't just test here `if tld in settings.ALLOW_LINKS_FROM`?

Reply:
tld will be a value like "COM", so doing that would only allow us to whitelist top level domains, not domains at any level